scripts/branch_track_creation.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""This is a script for creating branches and tracks"""
+
+import json
+import logging
+import os
+import sys
+
+import requests
+import yaml
+from git import Repo
+
+GITHUB_API_URL = "https://api.github.com"
+DEFAULT_REPO_OWNER = "canonical"
+MAIN_BRANCH_NAMES = ["main", "master"]
+GITHUB_TOKEN_NAME = "KUBEFLOW_BOT_TOKEN"
+
+HEADERS = {"content-type": "application/vnd.github.v3+json"}
+logger = logging.getLogger(__name__)
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+def get_git_diff():
+    """() -> [str]
+
+    Return a list of file paths of files changed in the last commit
+    """
+    cur_repo = Repo.init(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+    diff = cur_repo.git.diff("HEAD~1..HEAD", name_only=True).split("\n")
+    return diff
+
+
+def get_modified_releases_dirs(git_diff_file_paths):
+    """([str]) -> set(str)
+
+    Takes list of file paths as input, returns a list of releases directory path.
+    """
+    release_dir_name = "releases"
```

Comment:
Given this is a hardcoded constant might as well put it on top with things like `DEFAULT_REPO_OWNER`